package.json

```diff
@@ -36,6 +41,8 @@
   },
   "devDependencies": {
```

Comment:
According to proposed changes above, we need to add `eslint` and `mocha as a dev deps here